pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -393,6 +393,9 @@ type PoolSpec struct {
 
 	// The mirroring statusCheck
 	StatusCheck MirrorHealthCheckSpec `json:"statusCheck"`
+
+	// The quota settings
+	Quotas QuotaSpec `json:"quotas"`
```

Comment:
```suggestion
	Quotas *QuotaSpec `json:"quotas"`
```

Referencing https://github.com/rook/rook/pull/7264#discussion_r578814266, if we make this a pointer, the default is `nil`, and we can take that to mean that Rook should not modify existing quotas on the pool.

This is invalid yaml.
```yaml
quotas:
```

This means (implicitly) manage quotas, but ensure they are unset:
```yaml
quotas: {}
```

This explicitly asks Rook to manage quotas and unset them:
```yaml
quotas:
  maxBytes: 0
  maxObjects: 0
```